examples/promise-co-await/co.js

```diff
@@ -1,8 +1,8 @@
-var mysql = require('../promise.js');
+var mysql = require('../../promise.js');
```

Comment:
I think these examples are meant to be copied when user have installed the `node-mysql2` package. So please go ahead and commit the change as discussed